src/mame/video/model3.cpp

```diff
@@ -141,8 +102,8 @@ void model3_state::model3_exit()
 	fclose(file);
 #endif
 
-//  invalidate_texture(0, 0, 0, 6, 5);
-//  invalidate_texture(1, 0, 0, 6, 5);
+	invalidate_texture(0, 0, 0, 6, 5);
+	invalidate_texture(1, 0, 0, 6, 5);
```

Comment:
Why uncomment these things?  Presumably there’s a reason they were commented out.

Reply:
They were commented out because they were unnecessary, since `auto_alloc_array` was being used. Now that it's not, they're necessary to sweep the texture cache at exit time and free any cached textures.